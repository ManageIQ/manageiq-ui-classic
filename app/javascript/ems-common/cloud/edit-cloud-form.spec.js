require('angular-mocks');


describe('Ems edit cloud component', () => {
  let $scope;
  let element;

  beforeEach(() => {
    angular.module('ManageIQ');
  });

  beforeEach(inject(($compile, $rootScope) => {
    $scope = $rootScope;
    $scope.afterGet = true;
    element = $compile('<ems-edit-cloud-form test="test"></ems-edit-cloud-form>')($scope);
    $scope.$digest();
  }));

  it('should display name', () => {
    console.log('element: ', element.find('form'));
    expect(element).toBeDefined();
    expect(element).toMatchSnapshot();
  });
});
