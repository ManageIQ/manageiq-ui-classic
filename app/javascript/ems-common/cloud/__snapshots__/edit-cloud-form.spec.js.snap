// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ems edit cloud component should display name 1`] = `
Object {
  "0": <ems-edit-cloud-form
    class="ng-scope"
    test="test"
  />,
  "length": 1,
}
`;
